tests/interfaces_test.py

```diff
@@ -49,3 +49,7 @@ def test_save_config():
 def test_version():
     v = central.version()
     assert(v['version'] == '1.7.5.2-SNAPSHOT')
+
+def test_version():
```

Comment:
`test_version()` method already defined in line 49

Reply:
Probably due to a copy-paste. Fixed in 66824312e82091501f8d63a209cc1e8a6a6fc147.